src/utils/schemaLoaders.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const jsyaml = require('js-yaml');
+
+const cwd = process.cwd();
+
+const file = function (refValue, options) {
```

Comment:
any reason to make `file` an arrow function and not just a function?
Also isn't using noun for a function name somewhat confusing?

Reply:
changed